app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job so far. Please ensure that the application asks for confirmation of email.
 `:confirmable` is missing in your user model. Please fix that. it helps sends emails with confirmation instructions and verifies whether an account is already confirmed during sign in